frontend/src/components/transactions/__tests__/common.tsx

```diff
@@ -123,7 +123,7 @@ export const TRANSACTIONS: Transaction[] = [
             receiver_id: "receiver.test",
           },
         ],
-        status: { SuccessValue: null },
+        status: { SuccessValue: "" },
```

Comment:
Let's leave `null` in one of the tests

Reply:
But the types don't provider `null` as value, how come `null` appears here?